boston_data_test.go

```diff
@@ -0,0 +1,530 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat
+
+import "github.com/gonum/matrix/mat64"
+
+// Boston Housing Data of Harrison and Rubinfeld (1978)
+// http://dx.doi.org/10.1016/0095-0696(78)90006-2
+// http://lib.stat.cmu.edu/datasets/boston
+// Columns are: Per capita crime rate by town,
+//              Proportion of non-retail business acres per town,
```

Comment:
This doesn't show up in the godoc, so it won't be a problem, but rather than this (which will be wrong if it moves somewhere where it does get rendered), use something along the lines of

```
// Columns are;
//  per capita crime rate by town,
//  proportion of non-retail business acres per town,
//  nitric oxide concentration (parts per 10 million),
//  weighted distances to Boston employment centres,
//  index of accessibility to radial highways,
//  pupil-teacher ratio by town,
//  proportion of blacks by town,
//  average number of rooms per dwelling,
//  proportion of owner-occupied units built prior to 1940,
//  full-value property-tax rate per $10000,
//  median value of owner-occupied homes in $1000s.
```

(also note that this is now a sentence).